find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
+  else
+    max = array[0]
+    index = 0
+    while index < max_index
+      if max < array[index]
```

Comment:
I believe we need to switch the operator to `>` here.

```suggestion
      if max > array[index]
```

Reply:
The original line 12 was correct. You want to set the new max if the current max is less than the value.